docker/docker_image.go

```diff
@@ -40,20 +43,14 @@ func (i *dockerImage) RawManifest(version string) ([]byte, error) {
 }
 
 func (i *dockerImage) Manifest() (types.ImageManifest, error) {
-	// TODO(runcom): unused version param for now, default to docker v2-1
-	m, err := i.getSchema1Manifest()
-	if err != nil {
+	if err := i.retrieveRawManifest(); err != nil {
```

Comment:
you're totally right - I just overlooked thinking about this but it does smell a lot - thanks for the suggestion